src/migrations/20230630080126-delete-deprecated-permissions.js

```diff
@@ -0,0 +1,20 @@
+exports.up = function (db, cb) {
+    db.runSql(
+        `
+        DELETE FROM permissions WHERE permission IN ('CREATE_API_TOKEN', 'UPDATE_API_TOKEN', 'DELETE_API_TOKEN', 'READ_API_TOKEN');
+        DELETE FROM permissions WHERE permission = 'UPDATE_ROLE';
+        DELETE FROM permissions WHERE permission IN ('CREATE_ADMIN_API_TOKEN', 'UPDATE_ADMIN_API_TOKEN', 'DELETE_ADMIN_API_TOKEN', 'READ_ADMIN_API_TOKEN');
+
+        DELETE FROM role_permission rp where NOT EXISTS (SELECT * FROM permissions WHERE id = rp.permission_id);
```

Comment:
Ah, no cascade here. Did you check the statement locally?

Reply:
Yep, ran this locally, seemed to work as intended.

Reply:
I wrote this one a week ago because we were leaving the role_permissions linked to unexisting permissions. I used it locally to clean up my DB from those orphans. Is there anything else you'd like to check that I might be missing?

Reply:
I think this looks good.